plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -161,21 +185,33 @@ export function WriteItForMeButtonContainer() {
 			includedProps.push( name );
 		} );
 
-		return [
+		const instructions = [
 			`Compose an engaging product description for a product named "${ productName.slice(
 				0,
 				MAX_TITLE_LENGTH
-			) }".`,
-			...productPropsInstructions,
-			'Identify the language used in the product name, and craft the description in the same language.',
-			`Ensure the description is concise, containing no more than ${ DESCRIPTION_MAX_LENGTH } words.`,
-			'Structure the content into paragraphs using <p> tags, and use HTML elements like <strong> and <em> for emphasis.',
-			'Only if appropriate, use <ul> and <li> for listing product features.',
-			`Avoid including the properties (${ includedProps.join(
-				', '
-			) }) directly in the description, but utilize them to create an engaging and enticing portrayal of the product.`,
+			) }."`,
```

Comment:
I think we lost all of the category/attribute data here from the `productPropsInstructions` variable above? It was previous injected into the prompt here somewhere.

Reply:
Ahh, very good catch! There was a messy merge conflict from the prompt being updated in the settings PR so looks like some stuff got left out. Fixed in https://github.com/woocommerce/woocommerce/pull/39253/commits/65de7b59481dc51e0504538a253992b4418f0806